libraries/ui-shared/icons/TimeIcon.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
Please convert this file to use flow

Reply:
This file has no invariants. It just forward props ti Icon component